src/main/java/net/snowflake/client/core/arrow/ArrayConverter.java

```diff
@@ -1,13 +1,22 @@
+/*
+ * Copyright (c) 2024 Snowflake Computing Inc. All rights reserved.
+ */
 package net.snowflake.client.core.arrow;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.List;
 import net.snowflake.client.core.DataConversionContext;
+import net.snowflake.client.core.ObjectMapperFactory;
 import net.snowflake.client.core.SFException;
+import net.snowflake.client.jdbc.ErrorCode;
 import net.snowflake.client.jdbc.SnowflakeType;
 import org.apache.arrow.vector.complex.ListVector;
 
 public class ArrayConverter extends AbstractArrowVectorConverter {
 
   private final ListVector vector;
+  private static final ObjectMapper objectMapper = ObjectMapperFactory.getObjectMapper();
```

Comment:
What do you think about enabling `SerializationFeature.INDENT_OUTPUT`? This way it may be possible to have the same structure as from GS or not? I'm not convinced though, maybe we shouldn't depend on backend formatting.

Reply:
I am not sure if it is important, I wouldn't play with json formatting additionally